src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/Basic_Shapes.xaml.cs

```diff
@@ -0,0 +1,650 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.ConditionEvaluatesToConstant' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method:  ## Method file: Basic_Shapes.xaml.cs ## Method line: 380,3 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, SonarAnalyzer.SymbolicE](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5363624)